version.php

```diff
@@ -25,7 +25,7 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version   = 2024100300;        // The current plugin version (Date: YYYYMMDDXX).
+$plugin->version   = 2024103100;        // The current plugin version (Date: YYYYMMDDXX).
 $plugin->release   = 2024100300;
 $plugin->requires  = 2020061504;        // Requires this Moodle version.
```

Comment:
I think this is wrong, no? The initial version number for Moodle 4.1 is 2022112800 - so we probably need to update this.

Reply:
Updated :)